Tests/Unit/Model/PaymentInAdvancePaymentInformationTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+use Wirecard\Oxid\Model\PaymentInAdvancePaymentInformation;
+
+class PaymentInAdvancePaymentInformationTest extends OxidEsales\TestingLibrary\UnitTestCase
+{
+    const AMOUNT = "130";
```

Comment:
You don't need to add these values as consts if they are only used in one test.

Reply:
👍